### PR TITLE
java-oop first solution

### PR DESCRIPTION
**Pull request by imhatep**

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machine = new Machine[]{truck, bulldozer, excavator};
+        for (Machine machin : machine) {
```
Comment (petro-ushchuk) [inline src/main/java/core/basesyntax/MainApp.java:9]: Please, rename array to "machines" and for-each variable to "machine"

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: Java CI
 
-on: [push, pull_request]
+on: [ push, pull_request ]
 
 jobs:
   build:
```
Comment (petro-ushchuk) [inline .github/workflows/ci.yml:6]: Please, don't commit files doesn't required by task.

File: README.md [hunk 3]
```diff
@@ -6,4 +6,4 @@
 - Override `stopWork()` as well. It should print messages that certain machine stopped its work.
 - In `MainApp` class create `Machine` array with `Truck`, `Bulldozer` and `Excavator` and call methods `doWork()` and `stopWork()` in a loop.
 
-#### [Try to avoid these common mistakes, while solving task](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop)
+#### [Try to avoid these common mistakes, while solving task](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop)
```
Comment (petro-ushchuk) [inline README.md:9]: Please, don't commit files doesn't required by task.

File: pom.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
```
Comment (petro-ushchuk) [inline pom.xml:6]: Please, don't commit files doesn't required by task.

**Review by petro-ushchuk — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Machine.java [hunk 5]
```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public void doWork() {
+    }
+
+    public void stopWork() {
+    }
```
Comment (karma-o) [inline src/main/java/core/basesyntax/Machine.java:8]: these methods should be abstract and have no body

**Review by karma-o — CHANGES_REQUESTED** [review]
When  Petro said 
> do not commit the files that are not required by the task

that does not mean deleting them at all, we should fix it

**Review by karma-o — CHANGES_REQUESTED** [review]
there are still deleted configuration files

**Pull request closed without merging**